sdk/servicebus/Azure.Messaging.ServiceBus/tests/Processor/ProcessorLiveTests.cs

```diff
@@ -24,7 +24,7 @@ public async Task ProcessMessages(int numThreads, bool autoComplete)
                 enablePartitioning: false,
                 enableSession: false))
             {
-                await using var client = new ServiceBusClient(TestEnvironment.ServiceBusConnectionString);
+                await using var client = GetClient();
```

Comment:
nit:  This looks more like you're creating and disposing a client.   Maybe `CreateClient` would be a more accurate factory method name?